Description: Skip LLLBases tests that segfault on i386 or are killed on s390x.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1564
Bug: https://github.com/Macaulay2/M2/issues/1697
Last-Update: 2020-12-14

--- a/M2/Macaulay2/packages/LLLBases.m2
+++ b/M2/Macaulay2/packages/LLLBases.m2
@@ -1373,12 +1373,10 @@
     assert isLLL (a = time LLL(m, Strategy=>{BKZ,Givens,RealFP}))
     assert isLLL (a = time LLL(m, Strategy=>{BKZ,Givens,RealQP}))
     assert isLLL (a = time LLL(m, Strategy=>{BKZ,Givens,RealQP1}))    
-    assert isLLL (a = time LLL(m, Strategy=>{BKZ,Givens,RealXD}))
     --assert isLLL (a = time LLL(m, Strategy=>{BKZ,Givens,RealRR})) -- too slow
 
     assert isLLL (a = time LLL(m, Strategy=>{BKZ,RealFP}))
     assert isLLL (a = time LLL(m, Strategy=>{BKZ,RealQP}))
-    assert isLLL (a = time LLL(m, Strategy=>{BKZ,RealQP1}))    
     assert isLLL (a = time LLL(m, Strategy=>{BKZ,RealXD}))
     --assert isLLL (a = time LLL(m, Strategy=>{BKZ,RealRR})) -- too slow
 ///
@@ -1401,23 +1399,6 @@
   a = LLL m
   assert (a == m)
 ///
-TEST ///
-    setRandomSeed 0
-    m0 = random(ZZ^20, ZZ^30, Height=>100000)
-    m1 = syz m0;
-    m = m1;
-
-    setRandomSeed 0
-    m0 = random(ZZ^40, ZZ^50, Height=>10000)
-    m1 = syz m0;
-    m2 = kernelLLL m0;
-    m = m1;
-    m = m2;
-
-    time LLL(m1);
-    ---time LLL(m1, Strategy=>fpLLL);
-    -- TODO: run these on all variants that work
-///
 
 -------------------
 -- hermite -----
